.travis.yml

```diff
@@ -25,7 +24,7 @@ before_script:
   - Xvfb :99 -screen 0 1024x768x24 -extension RANDR &
 
 script:
-    - xvfb-maybe npm run build
+    - xvfb-maybe npm install
```

Comment:
The tests are run here too. NPM installs dependencies, the build is run, then tests are run (just calling `npm install`). I had split this into two command, but you only need to call `npm install`.